### PR TITLE
chore: remove unused `HomeHeroContainer`

### PR DESCRIPTION
**Pull request by dimatretyak**
### Description

This PR resolves []

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/Home/Home.tsx [hunk 1]
```diff
@@ -331,7 +331,7 @@ const Home = (props: Props) => {
                 return null
             }
           }}
-          ListHeaderComponent={HomeHeader}
+          ListHeaderComponent={<HomeHeader homePageAbove={homePageAbove} />}
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/app/Scenes/Home/Home.tsx:334]: I think the Trove is displayed further below on the home screen. Not sure about the `HomeHeader` component...

[the remark this answers is not in the document]
Comment (dimatretyak) [reply src/app/Scenes/Home/Home.tsx:334]: > the Trove is displayed further below on the home screen

yes, you're right, and it works the same way. so I think we can safely remove it from the `HomeHeader` component. @MounirDhahri what do you think?

https://user-images.githubusercontent.com/3513494/193006147-6fba314b-454a-47e4-a671-36554a5f8c95.mp4

[the remark this answers is not in the document]
Comment (olerichter00) [reply src/app/Scenes/Home/Home.tsx:334]: I think so, yes 👍

[the remark this answers is not in the document]
Comment (dimatretyak) [reply src/app/Scenes/Home/Home.tsx]: Removed

**Review by olerichter00 — APPROVED** [review]
Looks good 🌟

**Pull request merged**